classes/module/Module.php

```diff
@@ -411,6 +412,17 @@ public function install()
             return false;
         }
 
+        // Check for override conflicts
+        $moduleOverrideVerifier = $this->get(ModuleOverrideVerifier::class);
```

Comment:
```suggestion
        $moduleOverrideChecker = $this->get(ModuleOverrideChecker::class);
```
`Verifier` sounds like an anglicism no?

Reply:
Also should this be inculded in the `enable` method? It depends if the override classes are generated only on install or also on enable? (it would make sense that they are done on both actions, it also means on disable the overrides should be cleaned)

Reply:
ah damn yes, should have been `verifyer` 🙈

Reply:
done 👍